zeppelin-web/src/components/navbar/navbar.controller.js

```diff
@@ -90,22 +90,32 @@ function NavCtrl ($scope, $rootScope, $http, $routeParams, $location,
 
     // for firefox and safari
     logoutURL = logoutURL.replace('//', '//false:false@')
-    $http.post(logoutURL).error(function () {
-      // force authcBasic (if configured) to logout
-      $http.post(logoutURL).error(function () {
+
+    let config = (process.env.PROD) ? {headers: { 'X-Requested-With': 'XMLHttpRequest' }} : {}
```

Comment:
what is `process.env.PROD`?

Reply:
I kept this since this: https://github.com/apache/zeppelin/pull/2463